Name Fix: "Unavailiable to Unavailable"

rahaaatul opened the pull request:
I have a question to you,

Where do i edit the `.inf` file for windows? Because I found lot of errors there, and needs to be fixed immediately.
## Errors & Fixes
### Edit Scheme.Reg
The `scheme.reg` inside `install.inf` should be like this (ignoring the unavailable fix):
```
[Scheme.Reg]
HKCU,"Control Panel\Cursors\Schemes","%SCHEME_NAME%",,"%10%\%CUR_DIR%\%default%,%10%\%CUR_DIR%\%help%,%10%\%CUR_DIR%\%work%,%10%\%CUR_DIR%\%busy%,%10%\%CUR_DIR%\%cross%,%10%\%CUR_DIR%\%ibeam%,%10%\%CUR_DIR%\%handwriting%,%10%\%CUR_DIR%\%unavailiable%,%10%\%CUR_DIR%\%vertical%,%10%\%CUR_DIR%\%horizontal%,%10%\%CUR_DIR%\%diagonal_1%,%10%\%CUR_DIR%\%diagonal_2%,%10%\%CUR_DIR%\%move%,%10%\%CUR_DIR%\%alternate%,%10%\%CUR_DIR%\%link%"
```

### Change Name
Change the name from **BreezeX-RoséPine Cursors** to **Rose Pine**, because this is the issue:

![image](https://github.com/rose-pine/cursor/assets/86149861/42738026-bc07-471a-88ea-34308a6e366a)

rahaaatul committed:
Name Fix: "Unavailiable to Unavailable"

mvllow commented:
Thank you for the PR!

I'm not familiar with Windows so won't be able to answer your other questions but perhaps someone in the Discord could help you :)

rahaaatul commented:
> Thank you for the PR!

Hey! Thank you for merging!

> I'm not familiar with Windows so won't be able to answer your other questions but perhaps someone in the Discord could help you :)

I've found fixes for the `install.inf` not installing cursors properly. `Scheme.reg` is the issue here, seems like when you auto generate it, it's makes that error. Here's the correct `Scheme`:
```
[Scheme.Reg]
HKCU,"Control Panel\Cursors\Schemes","%SCHEME_NAME%",,"%10%\%CUR_DIR%\%default%,%10%\%CUR_DIR%\%help%,%10%\%CUR_DIR%\%work%,%10%\%CUR_DIR%\%busy%,%10%\%CUR_DIR%\%cross%,%10%\%CUR_DIR%\%ibeam%,%10%\%CUR_DIR%\%handwriting%,%10%\%CUR_DIR%\%unavailable%,%10%\%CUR_DIR%\%vertical%,%10%\%CUR_DIR%\%horizontal%,%10%\%CUR_DIR%\%diagonal_1%,%10%\%CUR_DIR%\%diagonal_2%,%10%\%CUR_DIR%\%move%,%10%\%CUR_DIR%\%alternate%,%10%\%CUR_DIR%\%link%,%10%\%CUR_DIR%\%link%",%10%\%CUR_DIR%\%link%""
```

This will fix the `install.inf`.

And to fix Name display error in Mouse settings window, use this to fix the error in `[Strings]` in `install.inf` file:
```
[Strings]
CUR_DIR             = "Cursors\Rose Pine"
SCHEME_NAME         = "Rose Pine"
```

Here's the full fixed version, you can distribute in windows package without worry (Tested it myself:

For Rose Pine:
```
; ===========================================================
; Auto-Generated File, Modified!
; ===========================================================
; This file has been modified by @rahaaatul to fix fatal errors.
; visit: https://github.com/rahaaatul

; It was originally generated by a tool
; called 'clickgen'. For more information,
; visit: https://www.github.com/ful1e5/clickgen
; ===========================================================


; BreezeX-RoséPine Cursors
; Rosé Pine BreezeX cursors. https://www.github.com/ful1e5/BreezeX_Cursor

[Version]
signature="$CHICAGO$"

[DefaultInstall]
CopyFiles = Scheme.Cur
AddReg    = Scheme.Reg,Wreg

[DestinationDirs]
Scheme.Cur = 10,"%CUR_DIR%"

[Scheme.Reg]
HKCU,"Control Panel\Cursors\Schemes","%SCHEME_NAME%",,"%10%\%CUR_DIR%\%default%,%10%\%CUR_DIR%\%help%,%10%\%CUR_DIR%\%work%,%10%\%CUR_DIR%\%busy%,%10%\%CUR_DIR%\%cross%,%10%\%CUR_DIR%\%ibeam%,%10%\%CUR_DIR%\%handwriting%,%10%\%CUR_DIR%\%unavailable%,%10%\%CUR_DIR%\%vertical%,%10%\%CUR_DIR%\%horizontal%,%10%\%CUR_DIR%\%diagonal_1%,%10%\%CUR_DIR%\%diagonal_2%,%10%\%CUR_DIR%\%move%,%10%\%CUR_DIR%\%alternate%,%10%\%CUR_DIR%\%link%,%10%\%CUR_DIR%\%link%",%10%\%CUR_DIR%\%link%""

[Wreg]
HKCU,"Control Panel\Cursors",,0x00020000,"%SCHEME_NAME%"
HKCU,"Control Panel\Cursors",Help,0x00020000,"%10%\%CUR_DIR%\%help%"
HKCU,"Control Panel\Cursors",AppStarting,0x00020000,"%10%\%CUR_DIR%\%work%"
HKCU,"Control Panel\Cursors",Wait,0x00020000,"%10%\%CUR_DIR%\%busy%"
HKCU,"Control Panel\Cursors",crosshair,0x00020000,"%10%\%CUR_DIR%\%cross%"
HKCU,"Control Panel\Cursors",precisionhair,0x00020000,"%10%\%CUR_DIR%\%cross%"
HKCU,"Control Panel\Cursors",NWPen,0x00020000,"%10%\%CUR_DIR%\%handwriting%"
HKCU,"Control Panel\Cursors",Grab,0x00020000,"%10%\%CUR_DIR%\%move%"
HKCU,"Control Panel\Cursors",SizeAll,0x00020000,"%10%\%CUR_DIR%\%move%"
HKCU,"Control Panel\Cursors",UpArrow,0x00020000,"%10%\%CUR_DIR%\%alternate%"
HKCU,"Control Panel\Cursors",Hand,0x00020000,"%10%\%CUR_DIR%\%link%"
HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Runonce\Setup\","",,"rundll32.exe shell32.dll,Control_RunDLL main.cpl @0,1"

[Scheme.Cur]
Help.cur
Work.ani
Busy.ani
Cross.cur
Handwriting.cur
Move.cur
Alternate.cur
Link.cur
Unavailable.cur
Vertical.cur
Default.cur
Diagonal_1.cur
Horizontal.cur
IBeam.cur
Diagonal_2.cur

[Scheme.Txt]

[Strings]
CUR_DIR             = "Cursors\Rose Pine"
SCHEME_NAME         = "Rose Pine"
default             = "Default.cur"
help                = "Help.cur"
work                = "Work.ani"
busy                = "Busy.ani"
cross               = "Cross.cur"
ibeam               = "IBeam.cur"
handwriting         = "Handwriting.cur"
unavailable        	= "Unavailable.cur"
vertical            = "Vertical.cur"
horizontal          = "Horizontal.cur"
diagonal_1          = "Diagonal_1.cur"
diagonal_2          = "Diagonal_2.cur"
move                = "Move.cur"
alternate           = "Alternate.cur"
link                = "Link.cur"
```

For Rose Pine Dawn:
```
; ===========================================================
; Auto-Generated File, Modified!
; ===========================================================
; This file has been modified by @rahaaatul to fix fatal errors.
; visit: https://github.com/rahaaatul

; It was originally generated by a tool
; called 'clickgen'. For more information,
; visit: https://www.github.com/ful1e5/clickgen
; ===========================================================


; BreezeX-RoséPine Cursors
; Rosé Pine BreezeX cursors. https://www.github.com/ful1e5/BreezeX_Cursor

[Version]
signature="$CHICAGO$"

[DefaultInstall]
CopyFiles = Scheme.Cur
AddReg    = Scheme.Reg,Wreg

[DestinationDirs]
Scheme.Cur = 10,"%CUR_DIR%"

[Scheme.Reg]
HKCU,"Control Panel\Cursors\Schemes","%SCHEME_NAME%",,"%10%\%CUR_DIR%\%default%,%10%\%CUR_DIR%\%help%,%10%\%CUR_DIR%\%work%,%10%\%CUR_DIR%\%busy%,%10%\%CUR_DIR%\%cross%,%10%\%CUR_DIR%\%ibeam%,%10%\%CUR_DIR%\%handwriting%,%10%\%CUR_DIR%\%unavailable%,%10%\%CUR_DIR%\%vertical%,%10%\%CUR_DIR%\%horizontal%,%10%\%CUR_DIR%\%diagonal_1%,%10%\%CUR_DIR%\%diagonal_2%,%10%\%CUR_DIR%\%move%,%10%\%CUR_DIR%\%alternate%,%10%\%CUR_DIR%\%link%,%10%\%CUR_DIR%\%link%",%10%\%CUR_DIR%\%link%""

[Wreg]
HKCU,"Control Panel\Cursors",,0x00020000,"%SCHEME_NAME%"
HKCU,"Control Panel\Cursors",Help,0x00020000,"%10%\%CUR_DIR%\%help%"
HKCU,"Control Panel\Cursors",AppStarting,0x00020000,"%10%\%CUR_DIR%\%work%"
HKCU,"Control Panel\Cursors",Wait,0x00020000,"%10%\%CUR_DIR%\%busy%"
HKCU,"Control Panel\Cursors",crosshair,0x00020000,"%10%\%CUR_DIR%\%cross%"
HKCU,"Control Panel\Cursors",precisionhair,0x00020000,"%10%\%CUR_DIR%\%cross%"
HKCU,"Control Panel\Cursors",NWPen,0x00020000,"%10%\%CUR_DIR%\%handwriting%"
HKCU,"Control Panel\Cursors",Grab,0x00020000,"%10%\%CUR_DIR%\%move%"
HKCU,"Control Panel\Cursors",SizeAll,0x00020000,"%10%\%CUR_DIR%\%move%"
HKCU,"Control Panel\Cursors",UpArrow,0x00020000,"%10%\%CUR_DIR%\%alternate%"
HKCU,"Control Panel\Cursors",Hand,0x00020000,"%10%\%CUR_DIR%\%link%"
HKLM,"SOFTWARE\Microsoft\Windows\CurrentVersion\Runonce\Setup\","",,"rundll32.exe shell32.dll,Control_RunDLL main.cpl @0,1"

[Scheme.Cur]
Help.cur
Work.ani
Busy.ani
Cross.cur
Handwriting.cur
Move.cur
Alternate.cur
Link.cur
Unavailable.cur
Vertical.cur
Default.cur
Diagonal_1.cur
Horizontal.cur
IBeam.cur
Diagonal_2.cur

[Scheme.Txt]

[Strings]
CUR_DIR             = "Cursors\Rose Pine Dawn"
SCHEME_NAME         = "Rose Pine Dawn"
default             = "Default.cur"
help                = "Help.cur"
work                = "Work.ani"
busy                = "Busy.ani"
cross               = "Cross.cur"
ibeam               = "IBeam.cur"
handwriting         = "Handwriting.cur"
unavailable        	= "Unavailable.cur"
vertical            = "Vertical.cur"
horizontal          = "Horizontal.cur"
diagonal_1          = "Diagonal_1.cur"
diagonal_2          = "Diagonal_2.cur"
move                = "Move.cur"
alternate           = "Alternate.cur"
link                = "Link.cur"
```

I don't know how to add that fix in your repo, otherwise I'd do it.

mvllow commented:
Ah I see, appreciate the examples. Let's continue the discussion in #2
